src/lib/hardforks.ts

```diff
@@ -0,0 +1,42 @@
+import { CURRENT_MAINNET_HARDFORK } from './constants';
+
+export enum Hardforks {
+  Frontier,
+  Homestead,
+  DaoFork,
+  TangerineWhistle,
+  SpuriousDragon,
+  Byzantium,
+  Constantinople,
+  Petersburg, // This hard fork doesn't exist in ethereum/execution-specs
+  Istanbul,
+  MuirGlacier,
+  Berlin,
+  London,
+  ArrowGlacier,
+  GrayGlacier,
+  Paris,
+  Shanghai,
+}
+
+// Retrieve all the hard forks from the starting hard fork to the current mainnet hard fork.
+export const getHardforksFrom = (startingHardfork: Hardforks): string[] =>
+  getHardforksFromTo(startingHardfork, CURRENT_MAINNET_HARDFORK);
+
+// Retrieve an array of hardforks from a starting hardfork to an ending hardfork (exclusive).
+export const getHardforksFromTo = (start: Hardforks, end: Hardforks): string[] => {
+  if (start > end) {
+    console.error(
+      `Error: the starting hard fork ${Hardforks[start]} (index: ${start}) occured after the ending hard fork ${Hardforks[end]} (index: ${end}). Arguments are wrong or must have been reversed.`
+    );
+    return [];
```

Comment:
Let's throw an error instead of quietly failing, since in production we should never hit this error
```suggestion
    throw new Error(
      `Error: the starting hard fork ${Hardforks[start]} (index: ${start}) occured after the ending hard fork ${Hardforks[end]} (index: ${end}). Arguments are wrong or must have been reversed.`
    );
```